docs/server-worker-extra-args

```diff
@@ -0,0 +1,18 @@
+Add the following to you main.tf and modify the extra arguements to your liking;   
+  
+  k3s_extra_server_args = [
+    "--write-kubeconfig-mode 644",
+    "--kube-apiserver-arg default-not-ready-toleration-seconds=30",
+    "--kube-apiserver-arg default-unreachable-toleration-seconds=30",
+    "--kube-controller-arg node-monitor-period=20s",
+    "--kube-controller-arg node-monitor-grace-period=20s",
+    "--kubelet-arg node-status-update-frequency=5s",
+    #"--debug",
+    #"--oidc-issuer-url ",
+    #"--oidc-client-id",
+    #"--oidc-username-claim"
+  ]
+
```

Comment:
At the end, I would add a little note with a link on where to find documentation on these args. It would make it easier for the reader to find exactly what they need and understand what everything does 🙂 

Something like this would be good:

```markdown
> For a complete list of all k3s server options, refer to the [k3s.io documentation](https://rancher.com/docs/k3s/latest/en/installation/install-options/server-config/).
```